aaudio/echo/src/main/cpp/echo_audio_engine.cc

```diff
@@ -123,11 +123,13 @@ void EchoAudioEngine::stopStreams() {
   if (recordingStream_ != nullptr) {
```

Comment:
I see:  you mean AAudioStream_close() could not take nullptr, correct?  that makes sense. let me add that one in.  If AAudioStream_close() could take a nullptr and do nothing, that would be good. I thought it as the memory function like free(), delete, I think they could take a nullptr.  let me add the patch first, thanks